src/inc/quic_platform_winuser.h

```diff
@@ -740,8 +740,44 @@ typedef struct CXPLAT_SQE {
 #if DEBUG
     BOOLEAN IsQueued; // Debug flag to catch double queueing.
 #endif
+    HANDLE WcpEvent;      // Manual-reset event for wake packets
```

Comment:
Maybe we should consider having an "extension" to the SQE type so that we pay the price for these extra members only if we are going to use them.

Maybe something like:

```
struct CXPLAT_SQE_WCP {
    CXPLAT_SQE sqe;
    HANDLE Event;
    HANDLE WaitCompletionPacket;
}
```

Not fully sure if it is worth it or not.